docs/modules/servers/pages/demo.adoc

```diff
@@ -0,0 +1,13 @@
+= Server Demo
+:navtitle: Demo
+
+The James Demo Server sets up  a light-weight, in-memory mail server
```

Comment:
`in-memory` is false, docker do save stuff on the disk. Maybe "zero dependency"?

Reply:
Yes, I like "zero-dependency". Maybe "command-line" would also be appropriate?

I am trying to convey that it is to be used on the command line just for demo purposes. It is not intended to be set up as a "real" server.

Reply:
Changed to "disposable".